### DISCUSSION TITLE
How to smoothly migrate the sql database?

### DISCUSSION THREAD

**Discussion by viveknuna** [quartznet/quartznet#2233] [Q&A]
I’m using quartz.net library and using sql server for quartz related tables. My application is a multi-tenant application and for each tenant there is a separate databases. Each database has its own quartz tables.

now I am planning to have a single database. I want to have only one database. I have only one concern that there are many jobs which are already scheduled from long time in these databases. So how can I move to a single database without any loss of scheduled jobs/triggers?

Will it work if I copy all the quartz tables records to this new quartz database respective tables?

quartz.net version: 3.5.0

**Comment by viveknuna** [comment]
@lahma could you please help on this?

**Reply by lahma** [reply]
I haven't done this myself.

**Reply by viveknuna** [reply]
@lahma I have raised a bounty for this question https://stackoverflow.com/q/60417324/6527049
